internal/pkg/kernel/kernel.go

```diff
@@ -243,10 +243,13 @@ func parse(parameters string) (parsed Parameters) {
 	return parsed
 }
 
+// CmdLine exposed for testing purposes
```

Comment:
Not a big fan of leaking package vars into tests. An arg to the `func read` would be cleaner IMO.

Reply:
On second thought: https://github.com/talos-systems/talos/pull/1494/files#r346676068